src/cryptoadvance/specter/templates/includes/fee-selection.html

```diff
@@ -238,11 +238,20 @@ <h4 slot="title">{{ _("Subtract fees from amount") }}</h4>
         // Gets executed after fee initialisation
         initWithFees() {
             // this.feesSlider.min is set above via min_fee
-            this.feesSlider.max = Math.floor(this.fees["fastestFee"] * 1.4)
-            let sliderMin = this.fees["minimumFee"]
-            let sliderMax = this.fees["fastestFee"]
-            let average = Math.floor((sliderMin + sliderMax) / 2)
-            this.feesSlider.value = average
+            // For low fee environment
+            if (this.fees["fastestFee"] == 1) {
```

Comment:
I triggered this if condition manually, by setting `this.fees["fastestFee"] = 1` just above the if condition. However the slider  maximum is not at  1.5, but at 4.... don't understand why yet....